src/main/java/com/blackducksoftware/integration/hub/item/PolymorphicHubItemListService.java

```diff
@@ -0,0 +1,123 @@
+package com.blackducksoftware.integration.hub.item;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.restlet.data.Method;
+import org.restlet.resource.ClientResource;
+
+import com.blackducksoftware.integration.hub.exception.BDRestException;
+import com.blackducksoftware.integration.hub.exception.ResourceDoesNotExistException;
+import com.blackducksoftware.integration.hub.json.RuntimeTypeAdapterFactory;
+import com.blackducksoftware.integration.hub.rest.RestConnection;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.reflect.TypeToken;
+
```

Comment:
Yes, it was originallycalled HubItemListService. The thought behind changing it to Polymorphic is that the list items' types must share a common parent. But I imagine all HubLists have that characteristic, so the "Polymorphic" in the name may be pointless.